data_steward/analytics/cdr_ops/retraction_in_place_validation_sandbox.py

```diff
@@ -0,0 +1,305 @@
+                    if 'observation' in name:
+                        if 'observation_period' in table:
+                            domain_id = 'observation_period'
+                        else:
+                            domain_id = 'observation'
+                        break
+                    else:
+                        domain_id = name
+                        break
+
+        old_row_counts_queries_list.append(
+            table_row_counts_query.render(project=project_id,
+                                          dataset=dataset,
+                                          table_name=table,
+                                          count='old_minus_aian_row_count',
+                                          days='1',
+                                          domain_id=domain_id))
+
+        new_row_counts_queries_list.append(
+            table_row_counts_query.render(project=project_id,
+                                          dataset=dataset,
+                                          table_name=table,
+                                          count='new_row_count',
+                                          days='0',
+                                          domain_id=domain_id))
+
+    old_row_counts_union_all_query = '\nUNION ALL\n'.join(
+        old_row_counts_queries_list)
+
+    new_row_counts_union_all_query = '\nUNION ALL\n'.join(
+        new_row_counts_queries_list)
+
+    old_retraction_table_count_query = (
+        f'{rids_query}\n{old_row_counts_union_all_query}'
+        if is_deidentified == 'true' else
+        f'{pids_query}\n{old_row_counts_union_all_query}')
+
+    new_retraction_table_count_query = (
+        f'{rids_query}\n{new_row_counts_union_all_query}'
+        if is_deidentified == 'true' else
+        f'{pids_query}\n{new_row_counts_union_all_query}')
+
+    old_count = execute(client, old_retraction_table_count_query)
+    new_count = execute(client, new_retraction_table_count_query)
+
+    results = pd.merge(old_count,
+                       new_count,
+                       on=['table_id', 'source', 'domain'],
+                       how='inner')
+
+    conditions = [
+        (results['old_minus_aian_row_count'] == results['new_row_count']) |
+        (results['old_minus_aian_row_count'] is None) &
+        (results['new_row_count'] is None),
+        (results['old_minus_aian_row_count'] is not None) &
+        (results['new_row_count'] is None),
+        (results['old_minus_aian_row_count'] is None) &
+        (results['new_row_count'] is not None)
+    ]
+    table_count_status = ['OK', 'POTENTIAL PROBLEM', 'PROBLEM']
+    results['table_count_status'] = np.select(conditions,
+                                              table_count_status,
+                                              default='PROBLEM')
+    results = results.reindex(columns=[
+        'table_id', 'old_minus_aian_row_count', 'new_row_count',
+        'table_count_status', 'source', 'domain'
+    ])
```

Comment:
Can we also add this so that the results with problems always come at the bottom?
`    ]).sort_values(by='table_count_status')`